From 8512b5ce8c1a0d5818bca417ca2c10cbb99bf54c Mon Sep 17 00:00:00 2001
From: Sai Vemuri <svemuri@redhat.com>
Date: Thu, 11 Sep 2014 22:43:19 -0400
Subject: [infiniband] cxgb4: Update Kconfig to include Chelsio T5 adapter

Message-id: <1410475447-94392-61-git-send-email-svemuri@redhat.com>
Patchwork-id: 93852
O-Subject: [RHEL7.1 PATCH BZ 1124947 060/108] RDMA/cxgb4: Update Kconfig to include Chelsio T5 adapter
Bugzilla: 1124947
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Signed-off-by: Hariprasad Shenai <hariprasad@chelsio.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 7d0a73a40c5ceb7524aa6a43f108de0dd8dbe3f0)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/hw/cxgb4/Kconfig b/drivers/infiniband/hw/cxgb4/Kconfig
index d4e8983..23f38cf 100644
--- a/drivers/infiniband/hw/cxgb4/Kconfig
+++ b/drivers/infiniband/hw/cxgb4/Kconfig
@@ -1,10 +1,10 @@
 config INFINIBAND_CXGB4
- tristate "Chelsio T4 RDMA Driver"
+ tristate "Chelsio T4/T5 RDMA Driver"
  depends on CHELSIO_T4 && INET && (IPV6 || IPV6=n)
  select GENERIC_ALLOCATOR
  ---help---
-   This is an iWARP/RDMA driver for the Chelsio T4 1GbE and
-   10GbE adapters.
+   This is an iWARP/RDMA driver for the Chelsio T4 and T5
+   1GbE, 10GbE adapters and T5 40GbE adapter.
 
    For general information about Chelsio and our products, visit
    our website at <http://www.chelsio.com>.
-- 
1.7.1